docs/user-guide/quick-start/admin-gui.md

```diff
@@ -5,78 +5,189 @@ As an Admin, you have the ability to create and deactivate users, change users'
+
+
+### Managing groups 
+To manage groups in the interface, navigate to the section Groups (instructions described above) and select the group you want to manage. 
+
+#### Add New Members:
+- Click on the +New members button to add members to the group.
+- In the new window, you can select members from the list or use the search bar to find specific users.
+
+![New Member](quick-start-images/15AddMembers.png)
+*Add new members to the group by clicking on **+New members**. A new window will appear where you can select users from the list or you can use the search bar to find specific users, e.g., **New User 2024***
+
+**Confirm New Members:**
+- After selecting the members, click **Add member to confirm**. The group will now display the recently added members.
+
+![Confirm New User](quick-start-images/17ConfirmMembers.png)
+*Confirm the new members to add by clicking **Add member**. The recently created group, **Demo ODM** for this example, will now show the recently added member, **New User 2024**.*
+
+
+#### Remove members
+To remove a member from a group, follow these steps:
+
+**1. Select the Group:**
+- Navigate to the group from which you intend to remove a user.
+- In the new window, either scroll through the list of members or use the search bar to find specific users.
+
+**2. Remove the Member:**
+- Click on the three dots next to the member's name and select **Remove member**.
+
+
+**Important Note**: Only members designated as **Group members** can be removed. If the user you intend to remove is an administrator, you must first ensure that another member is assigned as the group administrator, as each group must have at least one administrator. If you want to remove a user with the role of admin, you need to assign another administrator within the group.
+
+![Remove Members](quick-start-images/18RemoveMembers.png)
+***Remove members**. Select the group, such as **Demo ODM version 2**, from which you want to remove members. Click on the three dots next to the member's name and select "Remove member." Remember, every group requires at least one group administrator. If removing an admin, ensure another administrator is assigned beforehand.*
+
+
+#### Edit Group members:
+- You can edit the roles of group members by clicking on their role descriptions. You can change members to administrators or vice versa. Note that each group must have at least one administrator.
+
+![Edit Group Members](quick-start-images/19EditMembers.png)
+***Edit group members**. You can change the roles for the members to either **group members** or **group administrators**. Keep in mind that each group requires at least one admin.*
 
 ### Delete Groups
-!!! warning "To add, update, and delete groups "Manage groups" permission is required."
```

Comment:
We should keep this warning and add it everywhere for Groups OR Add it at the very beginning of Groups subsection